### PR TITLE
feat(text): add vertical align to Text and Image

### PR DESCRIPTION
**Pull request by jasmine-q**
## Description
Adds vertical align to Text and Image

💁‍

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [x] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Angus](https://github.com/angusfretwell), [Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), or [Long](https://github.com/KieraDOG)).
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [x] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

**Comment by jasmine-q** [comment]
> LGTM but did you need to add it to Image, too?

just added it, pls check again

File: src/components/Image/Image.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 import { css } from '@emotion/core';
-import { space, size, height, width } from 'styled-system';
+import { space, size, height, width, textAlign } from 'styled-system';
```
Comment (angusfretwell) [inline src/components/Image/Image.js]:
```suggestion
import { space, size, height, width, verticalAlign } from 'styled-system';
```

File: src/components/Image/Image.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 const Image = styled.img`
   display: block;
   max-width: 100%;
   height: auto;
 
-  ${space} ${size} ${height} ${width} ${props =>
+  ${space} ${size} ${height} ${width} ${textAlign} ${props =>
```
Comment (angusfretwell) [inline src/components/Image/Image.js]:
```suggestion
  ${space} ${size} ${height} ${width} ${verticalAlign} ${props =>
```

File: src/components/Image/Image.js [hunk 3]
```diff
@@ -20,6 +20,7 @@ Image.propTypes = {
   ...size.propTypes,
   ...height.propTypes,
   ...width.propTypes,
+  ...textAlign.propTypes,
```
Comment (angusfretwell) [inline src/components/Image/Image.js]:
```suggestion
  ...verticalAlign.propTypes,
```

File: src/components/index.d.ts [hunk 4]
```diff
@@ -305,7 +305,8 @@ declare module 'roo-ui/components' {
       SS.LineHeightProps,
       SS.SpaceProps,
       SS.TextAlignProps,
-      SS.TextStyleProps {
+      SS.TextStyleProps,
+      SS.VerticalAlignProps {
     hidden?: boolean;
   }
   export interface TextProps
```
Comment (angusfretwell) [inline src/components/index.d.ts]: Please add vertical align props to image interface

**Pull request merged**